roles/ansible_service_broker/tasks/install.yml

```diff
@@ -193,12 +193,6 @@
       path: /tmp/dcout
       data: "{{ lookup('template', 'asb_dc.yaml.j2') | from_yaml }}"
 
-- name: set auth name and type facts if needed
-  set_fact:
-    ansible_service_broker_registry_auth_type: "secret"
-    ansible_service_broker_registry_auth_name: "asb-registry-auth"
-  when: ansible_service_broker_registry_user != "" and ansible_service_broker_registry_password != ""
```

Comment:
I'm not sure this is actually equivalent, the previous behavior would default those to empty strings if the username/password aren't provided, and it wouldn't attempt to authorize with a secret. Now I think it will always try to authorize with the secret by default. I haven't touched this code in a while so let me know if this is just a distinction without a difference, or if the change in behavior is desired.